### ISSUE TITLE
PMUI Ctrl+C fails to copy in 16.?.x but worked in 15.x.x

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#8767]
Ctrl+C has been broken in certain scenarios in the PMUI.

The **Project** "Manage NuGet Packages" PMUI does not copy any text when using the Ctrl+C keystroke after selecting data from the description (date, package name, dependency name, version, etc). Right-click Copy does work.

**Solution** "Manage NuGet Packages" PMUI works correctly and copies with Ctrl+C.

VS2017 15.9.17 confirmed that it does work as expected.

Tested the following (broken):
16.4.0 Preview 2.0
16.4.0 Preview 3.0
16.3.6

Worked before? If so, with which NuGet version: 
Worked correctly on:
16.2.0 (VisualStudio.16.Release/16.2.0+29123.88)
15.9.17

## Detailed repro steps so we can see the same problem

1. Right click a project, click Manage NuGet Packages. 

2. Use any tab (Browse/installed/updates) and select a package.

3. In the description (right pane), select some text such as the Version number of this package or the Description.

4. Use Control + C keystroke on keyboard.

5. Try pasting into Notepad, and you'll see that nothing was copied.

6. Repeat step 3 and this time Right-click the selected text, and click "Copy".

7. Repeat step 5 and you will see the text was copied as expected.

**Comment by donnie-msft** [comment]
Related to previous work maybe?
https://github.com/NuGet/Home/issues/8324

**Comment by donnie-msft** [comment]
Inconsistent repro. New projects seem resilient to the bug.

NuGet.Client code - certain projects have this bug. Seems to be NuGet.Clients/ projects that are Class Libraries. ConsoleApps don't have this behavior? 
Definitely: **NuGet.Console** and **NuGet.SolutionRestoreManager** 

@dominoFire confirmed the issue on his machine with NuGet Client Solution.
